src/index.js

```diff
@@ -0,0 +1,55 @@
+// import _ from 'lodash';
+import './style.css';
+
+// function component() {
+//   const element = document.createElement('div');
+
+//   // Lodash, now imported by this script
+//   element.innerHTML = _.join(['Hello', 'webpack'], ' ');
+//   element.classList.add('hello');
+//   return element;
+// }
+
+// document.body.appendChild(component());
+
+const todoList = document.getElementById('todoList');
+// window.addEventListener('load', () => {
+//     todoList.style.display= "block";
+//     }
+// );
+
+const taskData = [
+  {
+    description: 'wash the dishes',
+    completed: true,
+    index: 1,
+  },
+  {
+    description: '  Complete To Do list project',
+    completed: true,
+    index: 2,
+  },
+  {
+    description: 'Attend family meeting',
+    completed: true,
+    index: 3,
+  },
+];
+
+let list = '';
+taskData.forEach((data) => {
+  list += `
+
+
+  <div id="${data.index}" class="checkContainer">
+  <div class="inputContainer">
+      <input id="checkBox" type="checkbox"  required ${data.index}>
```

Comment:
- [ ] **_Required:_** for best practices kindly remove `${data.index}` from the check box since it has no role and its not linked to any attribute